Scala/Scala.sublime-syntax

```diff
@@ -7,6 +7,9 @@ file_extensions:
   - sbt
 scope: source.scala
 variables:
+  # lookahead/behind for operators
+  oplookbehind: '(?<=[[[:alpha:]]0-9\s\(\)\[\]\{\}])'
```

Comment:
We don't use lookbehinds because it forces us to fallback to the slow oniguruma regex engine. Instead, push into a context whenever a construct could be valid.